pandas-stubs/io/clipboards.pyi

```diff
@@ -40,7 +40,7 @@ def read_clipboard(
     | None = ...,
     # squeeze: bool | None = ..., # deprecated: 1.4.0
     # prefix: str | None = ..., # deprecated: 1.4.0
-    mangle_dupe_cols: bool = ...,  # deprecated: 1.5.0
+    # mangle_dupe_cols: bool = ...,  # deprecated: 1.5.0
```

Comment:
Wouldn't mind removing the deprecation comments (on the other hand, typeshed adds deprecation comments)

Reply:
I've removed these.